# Do this patch after acconfig.h to avoid having autoconf regenerate the
# config.h.in file in case end user has autoconf...
diff -ur config.h.in-dist config.h.in
--- config.h.in-dist	Fri May  2 12:33:46 1997
+++ config.h.in	Thu May 15 22:46:22 1997
@@ -1,5 +1,11 @@
 /* config.h.in.  Generated automatically from configure.in by autoheader.  */
 
+/* Define if you have the <sys/param.h> header file.  */
+#undef HAVE_SYS_PARAM_H
+#ifdef HAVE_SYS_PARAM_H
+#include <sys/param.h>
+#endif
+
 /****** BEGIN USER CONFIGURATION SECTION *****/
 
 /*
@@ -38,9 +44,11 @@
  *  OpenBSD 2.x
  *  BSD/OS 2.x
  */
-/* #define BIND     1 */
-/* #define BSD42    1 */
-/* #define BSD44    1 */
+#if defined(BSD4_4)
+#define BIND     1
+#define BSD42    1
+#define BSD44    1
+#endif
 
 /*
  *  SCO 4.x
@@ -73,10 +81,13 @@
  * other programs which may modify your maildrops.
  * Currently you can only use one type.
  */
+#if defined(BSD44)
+#define FCNTL_LOCKING 1
+#else
 #define DOT_LOCKING   1
-/* #define FCNTL_LOCKING 1 */
 /* #define LOCKF_LOCKING 1 */
 /* #define FLOCK_LOCKING 1 */
+#endif
 
 /*
  * If you have defined DOT_LOCKING, then the default is to
